mayastor-test/test_csi.js

```diff
@@ -197,7 +198,13 @@ describe('csi', function() {
               let uuid = BASE_UUID + n;
               common.dumbCommand(
                 'publish_nexus',
-                { uuid: uuid, key: '' },
+                {
+                  uuid: uuid,
+                  key: '',
+                  // TODO: repeat this test for iSCSI and Nvmf
```

Comment:
I see a lot of TODO's lets remove the TODO's and add issues for this such that we can actually do them. Otherwise they get burried in the code and we will forget about them

Reply:
There is already a Jira (CAS-26) for Nvmf integration testing. I'll remove the TODO comments.

Reply:
Fixed by https://github.com/openebs/Mayastor/pull/125/commits/94d78ef5c45d51693a4f85c8f134259bf02ca7c3